olive/data/component/load_dataset.py

```diff
@@ -23,7 +23,7 @@ def simple_dataset(input_data, label_cols=None, **kwargs):
 
 
 @Registry.register_dataset()
-def huggingface_dataset(data_name=None, subset=None, split="validation", **kwargs):
+def huggingface_dataset(data_name=None, subset=None, split="validation", data_files=None, **kwargs):
```

Comment:
I was thinking that huggingface `load_dataset` has many arguments for specific case. Then I add kwargs, expose `data_files` is okay for me. 

But is it possible to get it from kwargs? We can update the documents to tell users that they can set any arguments from `load_dataset`

Reply:
Can you share an example for how to provide kwargs to load_dataset? I don't see it in the documents.

Reply:
Because of how `params_config` is used here https://github.com/microsoft/Olive/blob/main/olive/data/config.py#L98, `kwargs` is empty for all functions. I think it will only be non-empty if the `component.params` is provided specifically https://github.com/microsoft/Olive/blob/main/olive/data/config.py#L23

Reply:
As discussed offline, I have created a work item and will look at simplifying kwargs in a followup PR.